protocols/v2/subprotocols/template-distribution/README.md

```diff
@@ -0,0 +1,20 @@
+# template_distribution_sv2
+
+[![crates.io](https://img.shields.io/crates/v/common_messages_sv2.svg)](https://crates.io/crates/common_messages_sv2)
+[![docs.rs](https://docs.rs/common_messages_sv2/badge.svg)](https://docs.rs/common_messages_sv2)
+[![rustc+](https://img.shields.io/badge/rustc-1.75.0%2B-lightgrey.svg)](https://blog.rust-lang.org/2023/12/28/Rust-1.75.0.html)
+[![license](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)](https://github.com/stratum-mining/stratum/blob/main/LICENSE.md)
+[![codecov](https://codecov.io/gh/stratum-mining/stratum/branch/main/graph/badge.svg)](https://app.codecov.io/gh/stratum-mining/stratum/tree/main/protocols%2Fv2%2Fcommon_messages_sv2)
+
+`template_distribution_sv2` is a Rust crate that implements the set of messages used by multiple Stratum V2 roles.
```

Comment:
Isn't this a bit too much generic? It's the same of `common_messages_sv2`, but maybe you still need to work on the README. In any case, I think here the description should contain some reference to template distribution protocol at least.